fs/littlefs/lfs_vfs.c

```diff
@@ -1121,14 +1121,7 @@ static int littlefs_rewinddir(FAR struct inode *mountpt,
 }
 
 /****************************************************************************
- * Name: littlefs_bind
- *
- * Description: This implements a portion of the mount operation. This
- *  function allocates and initializes the mountpoint private data and
- *  binds the driver inode to the filesystem private data. The final
- *  binding of the private data (containing the driver) to the
- *  mountpoint is performed by mount().
- *
+ * Name: littlefs_read_block
```

Comment:
@Zhangshoukui since the function name and the description were wrong and you changed the function name, could you please include a proper Description too?